src/BulletCollision/Gimpact/btBoxCollision.h

```diff
@@ -234,7 +234,24 @@ ATTRIBUTE_ALIGNED16	(class) btAABB
 
 	btAABB()
 	{}
-
+	
```

Comment:
i think that this operator don't have much sense, but 

GIM_BVH_DATA and GIM_BVH_TREE_NODE have btAABB. and they be used in 

btAlignedObjectArray<GIM_BVH_TREE_NODE> that have

```
		class less
		{
			public:

				bool operator() ( const T& a, const T& b ) const
				{
					return ( a < b );
				}
		};
```

So it code work i add operator compare in GIM_PAIR, GIM_BVH_DATA, GIM_BVH_TREE_NODE, and btAAB.